project/Build.scala

```diff
@@ -501,6 +502,7 @@ object Build {
       libraryDependencies ++= Seq(
         "org.scala-lang.modules" % "scala-asm" % "7.0.0-scala-1", // used by the backend
         Dependencies.`compiler-interface`,
+        "com.googlecode.java-diff-utils" % "diffutils" % "1.3.0", // for SemanticdbTests
```

Comment:
We already have `dotty.tools.dotc.util.DiffUtil` class that we use for diffs in all the other tests. We should avoid bringing in external dependencies into the codebase.